src/main/java/org/springframework/integration/dsl/IntegrationFlowDefinition.java

```diff
@@ -2346,7 +2347,7 @@ public void accept(GenericEndpointSpec<BridgeHandler> bridge) {
 			}
 		}
 		if (hasSubFlows) {
-			channel(afterRouterChannel);
+			handle(bridgeHandler);
```

Comment:
Not critical, I suppose, but perhaps a future enhancement to lazily create this bridge, if needed, during `get()`.